spec/policies/placements/user_membership_policy_spec.rb

```diff
@@ -0,0 +1,67 @@
+require "rails_helper"
+
+RSpec.describe Placements::UserMembershipPolicy do
+  subject(:user_membership_policy) { described_class }
+
+  let(:user_membership) { create(:user_membership, user:, organisation:) }
+
+  permissions :destroy?, :remove? do
+    context "when current user and user are the the same user" do
```

Comment:
```suggestion
    context "when current user and user are the same" do
```

Or, potentially easier to understand:

```suggestion
    context "when the current user tries to delete themselves" do
```